src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,106 @@
+package duke;
+
+import java.util.Scanner;
+
+import exception.DukeException;
+import task.Task;
+
+/**
+ * Class that handles interactions with the user
+ */
+public class Ui {
+    private String line = "____________________________________________________________";
+    private String welcomeMessage = "Hello! I'm Duke\n What can I do for you?";
+    private Scanner sc = new Scanner(System.in);
```

Comment:
I am not too sure how common a practice it is to name a `Scanner` object "sc", but might it be better to use a more descriptive name such as "scanner" or "scannerObject" (especially since it is not referenced until line 70)?